src/classes/UTIL_String.cls-meta.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ApexClass xmlns="urn:metadata.tooling.soap.sforce.com" fqn="UTIL_String">
```

Comment:
I've never noticed an `<apexClass> tag with this format namespace string and and `fqn`. None of our other classes have that. It appears to be referencing the tooling api instead of the soap api. Honestly not sure what to make of this. @davisagli and @cdcarter What do you think about this?

Reply:
I initially deployed this using VS Code and SFDX: Create Apex Class function.  Seems like it used the tooling API.  Not sure why but I'll check on it.

Reply:
https://developer.salesforce.com/docs/atlas.en-us.api_tooling.meta/api_tooling/reference_objects_overview.htm defines the `metadata.tooling.soap.sforce.com` namespace as "Objects and types that occur in both the Metadata API WSDL and the Tooling API WSDL." Given that, I think the platform must be treating them similarly. CumulusCI's code to process -meta.xml files is careful to not assume a particular xml namespace and this appears to have deployed successfully for CI via the metadata API, so I don't think it matters too much which one is used.

I assume `fqn` stands for fully-qualified name but I can't find any documentation of what it is used for here!

Reply:
Thanks @davisagli. I guess we can leave it. I suppose in the end, it's how the file exists in the dev/packaging org that matters, not how the tooling or metadata/soap api retrieves and formats the meta-xml file. Since the file deploys correctly and to the best of my knowledge there is nothing in the Salesforce UI that allows that part of the meta-xml file to be viewed or modified, I'm cool with it.

@dvenor Maybe you're just the first dev to create & retrieve a new class using VSCode.